setup-scripts/blocklist-airtable.py

```diff
@@ -6,17 +6,24 @@
 
 import traceback
 import os
+import sys
 import re
 import asyncio
 import requests
 import json
 
 setup()
 
+
 AIRTABLE_API_KEY = os.getenv("AIRTABLE_API_KEY")
-AIRTABLE_BASE = os.getenv("AIRTABLE_BASE", "app89plJvA9EqTJEc")
-AIRTABLE_TABLE = os.getenv("AIRTABLE_TABLE", "Table%201")
-AIRTABLE_FIELD = os.getenv("AIRTABLE_FIELD", "Link")
+AIRTABLE_BASE = os.getenv("AIRTABLE_BASE")
+AIRTABLE_TABLE = os.getenv("AIRTABLE_TABLE")
+AIRTABLE_FIELD = os.getenv("AIRTABLE_FIELD")
+
+# Check environment variables are defined
+for e in [AIRTABLE_API_KEY, AIRTABLE_BASE, AIRTABLE_TABLE, AIRTABLE_FIELD]
+    if not e:
+        sys.exit("Configuration error: Missing AirTable environment variable.")
```

Comment:
really minor but lets use a meaningful variable name

```suggestion
for value in [AIRTABLE_API_KEY, AIRTABLE_BASE, AIRTABLE_TABLE, AIRTABLE_FIELD]
    if not value:
        sys.exit("Configuration error: Missing AirTable environment variable.")
```